R/centroids.R

```diff
@@ -355,8 +355,8 @@ setMethod(
   signature = c(x = 'Centroids', y = 'SpatialPolygons'),
   definition = function(x, y, returnList = FALSE, fn = NULL, ...) {
     return(over(
```

Comment:
Need to check for `sf`
```suggestion
  definition = function(x, y, returnList = FALSE, fn = NULL, ...) {
    check_installed(pkg = 'sf')
    return(over(
```